includes/migration-guide/runtime/core/systemthreadingtaskstask-no-longer-throw-objectdisposedexception-after.md

```diff
@@ -13,3 +13,5 @@ Be aware that Task methods may no longer throw <xref:System.ObjectDisposedExcept
 | Scope   |Minor|
 |Version|4.5|
 |Type|Runtime|
+
+<!-- TODO: Affected APIs? --> 
```

Comment:
@gewarren, I'll cleanup this TODOs. The issue now is what to put under `#### Affected APIs`? I'm not sure if it's "None" in all the cases where the heading is missing. It could be missing despite the existence of affected APIs. Do you know what should I do?

Reply:
@PriyaPurkayastha How important is it to list all affected APIs for the .NET Framework migration guide docs? I don't think we have the resources to go through each one and determine if there are any missing affected APIs.

Reply:
Priya and I spoke offline and agreed that at this point, we have to assume that the section was omitted because there are no affected APIs. So we should just put "Not detectable via API analysis".